toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -81,6 +82,11 @@ var defaultChrootEnv = []string{
 	"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
 }
 
+const (
+	isNotFinalAttempt = false
+	isFinalAttempt    = true
```

Comment:
IMHO, these constants make the code more confusing. `isFinalAttempt` sounds like a variable not a named constant. I would just use `true` and `false` in the code.

Reply:
I disagree. We've used to do only `true` and `false` in the code and we ended up not really knowing, what these values were setting without having to jump to the function's definition. Now we try to use well-named constants as convention across the code.